rust/arrow/src/util/display.rs

```diff
@@ -44,6 +44,66 @@ macro_rules! make_string {
     }};
 }
 
+macro_rules! make_string_interval_year_month {
+    ($column: ident, $row: ident) => {{
+        let array = $column
+            .as_any()
+            .downcast_ref::<array::IntervalYearMonthArray>()
+            .unwrap();
+
+        let s = if array.is_null($row) {
+            "0 years 0 mons 0 days 0 hours 0 mins 0.00 secs".to_string()
```

Comment:
same comment as below about NULL vs 0 handling

Reply:
👍 

Yes, it's better to represent NULL as string instead of it. Thanks